morituri/program/soxi.py

```diff
@@ -0,0 +1,47 @@
+import os
+
+from morituri.common import log, common
+from morituri.common import task as ctask
+
+SOXI = 'soxi'
+
+class AudioLengthTask(ctask.PopenTask, log.Loggable):
+    """
+    I calculate the length of a track in audio samples.
+
+    @ivar  length: length of the decoded audio file, in audio samples.
+    """
+    logCategory = 'AudioLengthTask'
+    description = 'Getting length of audio track'
+    length = None
+
+    def __init__(self, path):
+        """
+        @type  path: unicode
+        """
+        assert type(path) is unicode, "%r is not unicode" % path
+
+        self._path = path
+        self.logName = os.path.basename(path).encode('utf-8')
+
+        self.command = [SOXI, '-s', self._path]
```

Comment:
I've tried to learn as much about morituri.common.task as possible; do we need to assign self._path = path here if we only use it once?